### PR TITLE
Fixes dynamic trending wikis

### PR DESCRIPTION
**Pull request by Adebesin-Cell**
Fixes dynamic trending wikis

Screenhots:
localhost_4000_ (6)

fixes https://github.com/EveripediaNetwork/issues/issues/1016

### CODE REVIEW

File: src/utils/HomepageUtils/getDate.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+export const formatDate = (dateString: number) => {
```
Comment (kesar) [inline src/utils/HomepageUtils/getDate.ts:1]: you dont need this function, you can use something like this:

  new Date().toLocaleDateString('en-US', {year: 'numeric', month: '2-digit', day: '2-digit'}), // 08/19/2020 (month and day with two digits)

Comment (Adebesin-Cell) [reply src/utils/HomepageUtils/getDate.ts:1]: Ha! Okay @kesar.

File: src/pages/index.tsx [hunk 2]
```diff
@@ -17,8 +22,10 @@ import RankingList from '@/components/Landing/RankingList'
 import { nftLisitngAPI } from '@/services/nftlisting'
 import { getNFTRanking, getTokenRanking, rankingAPI } from '@/services/ranking'
 import { Hero } from '@/components/Landing/Hero'
+import { getDateRange } from '@/utils/HomepageUtils/getDate'
 
 const RANKING_LIST_LIMIT = 10
+const TRENDING_WIKIS_AMOUNT = 5
```
Comment (kesar) [inline src/pages/index.tsx:28]: lets also add a const of number of days to calculate the days we want to track. Starting w 1 day. That way we can easily tweak it

**Comment by kesar** [comment]
besides those 2 small code suggestions its good to go, so feel free to merge once thats fixed 👍🏻

**Comment by Adebesin-Cell** [comment]
> besides those 2 small code suggestions its good to go, so feel free to merge once thats fixed 👍🏻

Okay @kesar.

File: src/utils/HomepageUtils/getDate.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+export const getDateRange = (dayRange = 1) => {
+  const startOfDay = new Date().setUTCHours(0, 0, 0, 0)
```
Comment (kesar) [inline src/utils/HomepageUtils/getDate.ts:2]:
```suggestion
  const startOfDay = new Date()
```

File: src/utils/HomepageUtils/getDate.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  const endOfDay =
+    new Date().setUTCHours(23, 59, 59, 999) + dayRange * 24 * 60 * 60 * 1000
```
Comment (kesar) [inline src/utils/HomepageUtils/getDate.ts:4]:
```suggestion
    new Date() - dayRange * 24 * 60 * 60 * 1000
```

File: src/utils/HomepageUtils/getDate.ts [hunk 5]
```diff
@@ -0,0 +1,19 @@
+export const getDateRange = (dayRange = 1) => {
+  const startOfDay = new Date()
```
Comment (kesar) [inline src/utils/HomepageUtils/getDate.ts:2]:
```suggestion
  const startOfDay = new Date() - dayRange * 24 * 60 * 60 * 1000
```

File: src/utils/HomepageUtils/getDate.ts [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  const endOfDay =
+    new Date() - dayRange * 24 * 60 * 60 * 1000
```
Comment (kesar) [inline src/utils/HomepageUtils/getDate.ts:4]:
```suggestion
    new Date()
```

**Comment by kesar** [comment]
ups sorry I messed up a bit the commits :_

**Comment by Adebesin-Cell** [comment]
@kesar I think it works fine now 👍.

**Review by kesar — APPROVED** [review]

**Comment by kesar** [comment]
awesome lets go!

**Pull request merged**